### ISSUE TITLE
Kill R_NOTNULL

### ISSUE THREAD

**Issue by radare** [radare/radare2#11553]
This is (void*)(size_t)-1UL, which is a bad idea and shouldnt be used at all

**Comment by davidpolverari** [comment]
Well, as I didn't understand before what this issue was about, I analyzed the places where R_NOTNULL is used. This is what I saw:

* There are different definitions of **`static void * load_bytes(RBinFile *bf, const ut8 *buf, ut64 sz, ut64 loadaddr, Sdb *sdb)`**, one in each file that deals with a different bin file.

* **R_NOTNULL** is used only as a return value to _some_  of those `load_bytes` definitions.

* In those cases, that function only calls **`static bool check_bytes(const ut8 *buf, ut64 length)`**, which, by its turn, only returns a _bool,_ and doesn't use all the parameters that were passed to `load_bytes`.

* Furthermore, it doesn't make sense to return a pointer, even more a constant one,  in those cases, if they are totally unrelated with the results of check_bytes.

So the question is: in those cases, what should `load_bytes` return? A NULL? Any other pointer will be completely arbitrary and, if used somewhere else, could lead to bad results.
